ItalianTranslation_PlanetCoaster.xml

```diff
@@ -195,7 +195,7 @@
 		<entry key="Blueprints_DragonBlueprint01" translation="Flight of the Dragon"/>
 		<entry key="Blueprints_DragonBlueprint01Desc" translation="Un layout per il Junior coaster"/>
 		<entry key="Blueprints_DragonSlayer" translation="A Dragon whispering scene"/>
-		<entry key="Blueprints_DragonSlayerDesc" translation="A Dragon whispering scene"/>
+		<entry key="Blueprints_DragonSlayerDesc" translation="Scenario del Dragone Sussurrante"/>
```

Comment:
Sei sicuro di questa? Controllata in game?

Reply:
Mo vedo.